src/test/java/co/rsk/federate/adapter/ThinConverterTest.java

```diff
@@ -13,18 +13,16 @@ class ThinConverterTest {
 
     private static int nhash = 0;
 
-    public static org.bitcoinj.core.Sha256Hash createOriginalHash() {
+    private org.bitcoinj.core.Sha256Hash createOriginalHash() {
```

Comment:
just a thought, could we just import this class? I think it adds unnecessary noise like this

IMHO, it will look much clearer to see `private Sha256Hash createOriginalHash()`

Reply:
One is creating a hash from bitcoinj-core and the other one from bitcoinj-thin. That's why the have the full package name, to distinguish them

Reply:
I agree. And it is already imported.